.github/workflows/domain_ci.yml

```diff
@@ -125,6 +125,9 @@ jobs:
       - name: Check out torchdata repository
         uses: actions/checkout@v2
 
+      - name: Install missing dependency
+        run: pip install requests
+
       - name: Install torchdata
         run: python setup.py install
```

Comment:
Thanks for fixing it. Can you move this part into `Install torchdata` with `pip install -r requirements.txt`?